container/sagemaker/tensorflow-serving.js

```diff
@@ -13,9 +13,68 @@ function invocations(r) {
     }
 }
 
-// ping check does not POST to a specific model, because MME starts with 0 model
+function mme_ping(r) {
```

Comment:
this is almost identical to ping_tfs_1_11 - we should refactor to avoid the duplicated code.

Reply:
Merged them together.